tests/test_grpc_server.py

```diff
@@ -59,6 +59,30 @@ def test_create_bucket(self):
         self.assertEqual(response.name, "projects/_/buckets/test-bucket-name")
         self.assertEqual(response.bucket_id, "test-bucket-name")
 
+    def test_delete_object(self):
+        media = b"The quick brown fox jumps over the lazy dog"
+        request = testbench.common.FakeRequest(
+            args={"name": "object-name"}, data=media, headers={}, environ={}
+        )
+        blob, _ = gcs.object.Object.init_media(request, self.bucket.metadata)
+        self.db.insert_object(request, "bucket-name", blob, None)
+        full_bucket_name = blob.metadata.bucket
+        context = unittest.mock.Mock()
+        _ = self.grpc.DeleteObject(
+            storage_pb2.DeleteObjectRequest(
+                bucket=full_bucket_name, object="object-name"
+            ),
+            context,
+        )
+        context = unittest.mock.Mock()
+        items, _ = self.db.list_object(
```

Comment:
I'm thinking it would be better to perform a GET request and receive a 404. Not super critical though.

Reply:
Ack. I am happy with the test as-is.  It does its job.  I would hope it returns a `NOT_FOUND` though, 404 is a http-ism.

Reply:
ah, yep lol; still need to switch my mindset it seems, thanks for clarifying!